packages/commercetools/api-client/src/api/customerResetPassword/index.ts

```diff
@@ -0,0 +1,35 @@
+import defaultQuery from './defaultMutation';
+import { ResetPasswordResponse } from 'src/types/Api';
+import { CustomQuery, Logger } from '@vue-storefront/core';
+import gql from 'graphql-tag';
+
+const customerResetPassword = async (context, tokenValue: string, newPassword: string, customQuery?: CustomQuery): Promise<ResetPasswordResponse> => {
+  const { locale, acceptLanguage } = context.config;
+  const defaultVariables = tokenValue && newPassword
+    ? {
+      locale,
+      acceptLanguage,
+      tokenValue,
+      newPassword
+    }
+    : { acceptLanguage };
+
+  const { customerResetPassword } = context.extendQuery(
+    customQuery, { customerResetPassword: { query: defaultQuery, variables: defaultVariables } }
+  );
+
+  try {
+    const result = await context.client.mutate({
```

Comment:
Same as in `customerCreatePasswordResetToken/index.ts`

Reply:
It is easier to locate a bug because it is logged with our VSF logger.